spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-zipkin-stream/pom.xml

```diff
@@ -54,6 +55,11 @@
 			<artifactId>spring-cloud-sleuth-zipkin-stream</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>io.zipkin</groupId>
```

Comment:
@marcingrzejszczak added this as otherwise the example won't show the UI (also latest zipkin-web is a dumb proxy to the query server, so deploying it would just point to 404 unless we host it)